crates/metadata/src/test.rs

```diff
@@ -65,3 +65,22 @@ pub enum WaveFormTarget {
     #[serde(rename = "directory")]
     Directory { path: PathBuf },
 }
+
+#[derive(Clone, Copy, Default, Serialize, Deserialize, PartialEq, Eq)]
+#[serde(tag = "type")]
+pub enum WaveFormFormat {
```

Comment:
How about adding `waveform_format: WaveFormFormat` to `Test`, and supporting `veryl test --wave` which uses default waveform format defined in `Veryl.toml`?
So user can use simply `veryl test --wave`, and can specify `veryl test --wave fst` as needs.

Reply:
I introcuded the new field in `Veryl.toml` but discarded my initial idea of specifying the target format via cli.
Is that ok for you?

Reply:
I added the field in the `Veryl.toml` but discarded my initial idea of specifiying it via cli.

Is that ok for you?